xml/System/Delegate.xml

```diff
@@ -96,7 +96,7 @@
   
  When a delegate represents an instance method closed over its first argument (the most common case), the delegate stores a reference to the method's entry point and a reference to an object, called the target, which is of a type assignable to the type that defined the method. When a delegate represents an open instance method, it stores a reference to the method's entry point. The delegate signature must include the hidden `this` parameter in its formal parameter list; in this case, the delegate does not have a reference to a target object, and a target object must be supplied when the delegate is invoked.  
   
- When a delegate represents a static method, the delegate stores a reference to the method's entry point. When a delegate represents a static method closed over its first argument, the delegate stores a reference to the method's entry point and a reference to a target object assignable to the type of the method's first argument. When the delegate is invoked, the first argument of the static method receives the target object.  
+ When a delegate represents a static method, the delegate stores a reference to the method's entry point. When a delegate represents a static method closed over its first argument, the delegate stores a reference to the method's entry point and a reference to a target object assignable to the type of the method's first argument. When the delegate is invoked, the first argument of the static method receives the target object. This first argument must be a reference type; not a boxed value type.
```

Comment:
> not a boxed value type

I think you should drop this part. Boxed value type acts like a type reference. It actually works fine 
```suggestion
 When a delegate represents a static method, the delegate stores a reference to the method's entry point. When a delegate represents a static method closed over its first argument, the delegate stores a reference to the method's entry point and a reference to a target object assignable to the type of the method's first argument. When the delegate is invoked, the first argument of the static method receives the target object. This first argument must be a reference type.
```
boxed value type acts like a reference type. It actually works fine with boxed value types. E.g. this is using boxed value type and it works just fine:
```
using System;
using System.Numerics;

static class TestClass
{
	public static float GetLength(object x) => ((Vector4)x).Length();
	
	static Func<float> StructStatic(Vector4 x)
	{
		var methodInfo = typeof(TestClass).GetMethod(nameof(GetLength));
		return (Func<float>)Delegate.CreateDelegate(typeof(Func<float>), x, methodInfo);
	}

	static void Main()
	{
		Vector4 x = default;
		Console.WriteLine(StructStatic(x)());
	}
}
```
>  potentially why this limitation exists?

Too complicated to handle this corner case in the runtime. Not worth it.
https://github.com/dotnet/runtime/blob/main/src/coreclr/vm/comdelegate.cpp#L2716

Reply:
Done @jkotas.